air/tests/test_module/issues/issue_300.rs

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use air_test_utils::prelude::*;
+
+#[test]
+// test for github.com/fluencelabs/aquavm/issues/300
+fn issue_300() {
+    let peer_id_1 = "peer_id_1";
+    let mut peer_vm_1 = create_avm(echo_call_service(), peer_id_1);
+    let peer_id_2 = "peer_id_2";
+    let mut peer_vm_2 = create_avm(echo_call_service(), peer_id_2);
+
+    let script = f!(r#"
```

Comment:
I've managed to rewrite it with testing-framework with the following script:

```
    let script = f!(r#"
        (new $stream
            (seq
               (par
                (call "{peer_id_1}" ("" "") [2] $stream)  ; behaviour=echo
                (call "{peer_id_2}" ("" "") [1] $stream)  ; behaviour=echo
               )
               (call "{peer_id_1}" ("" "") [0]) ; behaviour=echo
            )
        )
    "#);

    let executor = TestExecutor::simple(TestRunParameters::from_init_peer_id(peer_id_2), &script).unwrap();
    executor.execute_one(peer_id_2).unwrap();
    let result_1 = executor.execute_one(peer_id_1).unwrap();
    let actual_trace = trace_from_result(&result_1);

```

The `expected_trace` gets one extra value `executed_state::scalar_number(0)`.

Reply:
That's ok.